api/app/serializers/spree/v2/storefront/payment_serializer.rb

```diff
@@ -4,7 +4,7 @@ module Storefront
       class PaymentSerializer < BaseSerializer
         set_type :payment
 
-        attributes :amount, :response_code, :number, :cvv_response_code, :cvv_response_message,
+        attributes :amount, :response_code, :number, :state, :cvv_response_code, :cvv_response_message,
```

Comment:
Do we need to expose `state` to public?

Reply:
@VygovskySergey do we have any other way to know whether a payment is valid or not?

Reply:
Hm, good question. Why do you need to know if payment is valid or not?

Reply:
Uhm.. for many reasons. Invalid payments are not that useful for the end user, especially before order submitted.

It would be better to sort them out in this serializer, but I guess it's a bit out of scope fo this PR

Reply:
Than it is better just return some field like `valid`, because `state` is also meaningless for end user, if you are not into spree internals. @damianlegawiec what do you think?

Reply:
There's nothing wrong with sending payment `state` anyway IMHO

Reply:
@caudatecoder @VygovskySergey let's return only valid payments in this serializer, ok? And let's not expose `state`. Storefront API is a simplified set of APIs for the UI (JS/Mobile) and we don't necessarily need to bother API users with the state machine and so on. It should be super simple :)

Reply:
@damianlegawiec @VygovskySergey done ✅